tests/TestableIO.System.IO.Abstractions.TestingHelpers.Tests/MockFileMoveTests.cs

```diff
@@ -249,6 +249,26 @@ public async Task MockFile_Move_ShouldThrowNotSupportedExceptionWhenDestinationP
         await That(action).Throws<NotSupportedException>();
     }
 
+    [Test]
+    [WindowsOnly(WindowsSpecifics.Drives)]
+    public async Task MockFile_Move_CaseOnlyRename_ShouldChangeCase()
+    {
+        string sourceFilePath = @"c:\something\demo.txt";
+        string destFilePath = @"c:\something\DEMO.TXT";
+        string sourceFileContent = "content";
+
+        var fileSystem = new MockFileSystem(new Dictionary<string, MockFileData>
+        {
+        {
+                sourceFilePath, new MockFileData(sourceFileContent)}
+        });
+
+        fileSystem.File.Move(sourceFilePath, destFilePath);
+
+        await That(fileSystem.FileExists(destFilePath)).IsTrue();
```

Comment:
Also here, you would need to adjust the assertions to use the methods from `IFileSystem`:
```csharp
        await That(fileSystem.File.Exists(destFilePath)).IsTrue();
        await That(fileSystem.File.ReadAllText(destFilePath)).IsEqualTo(sourceFileContent);
```